deep_daze/deep_daze.py

```diff
@@ -404,7 +370,7 @@ def image_output_path(self, sequence_number=None):
         if self.save_date_time:
             current_time = datetime.now().strftime("%y%m%d-%H%M%S_%f")
             output_path = f"{current_time}_{output_path}"
-        return Path(f"{output_path}.png")
+        return Path(f"{output_path}.jpg")
```

Comment:
Why the switch to jpg?

Reply:
<img width="1067" alt="jpegsavingnotworking" src="https://user-images.githubusercontent.com/3994972/109230412-778f3c00-778a-11eb-92e0-a3a53550ae51.PNG">

The `simplified notebook` will need a small change if we're gonna go with JPEG. 

```python
from tqdm import trange
from IPython.display import Image, display

from deep_daze import Imagine

TEXT = 'worf getting attacked by a barrel' #@param {type:"string"}
NUM_LAYERS = 32 #@param {type:"number"}
SAVE_EVERY =  1#@param {type:"number"}
IMAGE_WIDTH = 512 #@param {type:"number"}
SAVE_PROGRESS = True #@param {type:"boolean"}
LEARNING_RATE = 1e-5 #@param {type:"number"}
ITERATIONS = 1050 #@param {type:"number"}

model = Imagine(
    text = TEXT,
    num_layers = NUM_LAYERS,
    save_every = SAVE_EVERY,
    image_width = IMAGE_WIDTH,
    lr = LEARNING_RATE,
    iterations = ITERATIONS,
    save_progress = SAVE_PROGRESS
)

for epoch in trange(20, desc = 'epochs'):
    for i in trange(ITERATIONS, desc = 'iteration'):
        model.train_step(epoch, i)

        if i % model.save_every != 0:
            continue

        filename = TEXT.replace(' ', '_')
        image = Image(f'./{filename}.png') # This needs to be jpg
        display(image)
```

Unfortunately the notebook isn't in source control, so @lucidrains will have to make that change.